server/src/test/java/org/apache/druid/server/coordination/ZkCoordinatorTest.java

```diff
@@ -67,9 +76,32 @@ public String getBase()
   };
   private ZkCoordinator zkCoordinator;
 
+  private File infoDir;
```

Comment:
Please use `TemporaryFolder` of JUnit (an example is found [here](https://github.com/apache/druid/blob/master/core/src/test/java/org/apache/druid/data/input/impl/LocalInputSourceTest.java#L49-L50)). This rule automatically cleans up the temp directory on exit.

Reply:
Changed this to use TemporaryFolder, I also changed SegmentLoadDropHandlerTest which is where I got the code from

Reply:
👍